### DISCUSSION TITLE
Discrepancy in Cumulative Billing due to multiple currencies

### DISCUSSION THREAD

**Discussion by mantewari** [cloud-carbon-footprint/cloud-carbon-footprint#1151] [Q&A] [answered]
The total billing is not taking currency into consideration, by default it is showing everything as USD however if the accounts are configured in a different currencies it shows incorrect data on the dashboard.

**Comment by ericksod** [comment, marked as answer]
Hi @mantewari 
Thanks for bringing this up.
For now we operate on the assumption that billing data is in one currency. We don't currently do any currency conversions but this is an interesting issue.
We'd have to think a bit more about how to go about this and all the implications (eg choosing a default currency, unknown estimations, etc.).
We will also in the short term look into how the UI is handling currency at the moment and fix if needed for the single currency behavior.
